plugins/outputs/orionres/processor/process.go

```diff
@@ -15,44 +15,25 @@ const (
 	// counterSuffix = "_cnt"
 )
 
+var (
+	dbSafetyLevelCommitted string = "committed"
```

Comment:
Suggest making this `const` if the level doesn't change.

Reply:
SafetlyLevel is a string pointer.  Using a const instead of var would be an extra alloc each call in the pointer conversion.